package.json

```diff
@@ -34,10 +34,13 @@
     "@semantic-release/release-notes-generator": "^10.0.3",
     "broken-link-checker-local": "^0.2.1",
     "cz-conventional-changelog": "^3.3.0",
+    "dotenv": "^16.0.0",
     "eleventy-plugin-dart-sass": "^1.0.3",
     "husky": "^7.0.4",
+    "isomorphic-fetch": "^3.0.0",
```

Comment:
`fetch` (for Node)

Reply:
`Fetch` is also included starting Node v17.5.0 🎊

Reply:
Yeah!

Reply:
Almost updated our repo default, but at this point it will only give everyone more work right now.

Reply:
`fetch` globally available, is also something I like in Astro :-) 
- https://docs.astro.build/en/guides/data-fetching/